src/blob/index/core.rs

```diff
@@ -59,14 +59,29 @@ struct MemoryAttrs<K> {
     marker: PhantomData<K>,
 }
 
+const BTREE_B_FACTOR: usize = 6;
+const BTREE_VALUES_LEN: usize = BTREE_B_FACTOR * 2 - 1;
+const BTREE_EDGES_LEN: usize = BTREE_B_FACTOR * 2;
+
 impl<K> MemoryAttrs<K>
 where
     for<'a> K: Key<'a>,
 {
     const KEY_SIZE: usize = K::LEN as usize;
-    const BPTREE_ENTRY_SIZE: usize =
+    const BTREE_ENTRY_SIZE: usize =
         size_of::<Vec<u8>>() + MemoryAttrs::<K>::KEY_SIZE + size_of::<Vec<RecordHeader>>();
     const RECORD_HEADER_SIZE: usize = size_of::<RecordHeader>() + MemoryAttrs::<K>::KEY_SIZE;
+    // Each node in BTreeMap contains preallocated vectors of 11 values and 12 edges.
+    // Although count of nodes can't be determined without reimplementing insertion algorithm,
+    // we can use approximation of overhead size added per one key
+    const BTREE_SIZE_MULTIPLIER: f64 = 
```

Comment:
I suggest to split this into 3 const:
1. `BTREE_LEAF_NODE_SIZE = ...`
2. `BTREE_INTERNAL_NODE_SIZE = BTREE_LEAF_NODE_SIZE + ...`
3. `BTREE_SIZE_MULTIPLIER = ...`